alloydb-jdbc-connector/src/test/java/com/google/cloud/alloydb/ConnectionInfoCacheTest.java

```diff
@@ -98,348 +92,4 @@ public void testGetConnectionInfo_returnsConnectionInfo() {
     List<X509Certificate> certificateChain = connectionInfo.getCertificateChain();
     assertThat(certificateChain).hasSize(2);
   }
-
-  @Test
-  public void testGetConnectionInfo_schedulesNextOperation() {
```

Comment:
I think we should keep the tests that start with `testGetConnectionInfo_` because they validate the ConnectionInfo behavior.

I agreed that the tests starting with `testForceRefresh_` are covered by the [RefresherTest.java](https://github.com/GoogleCloudPlatform/alloydb-java-connector/blob/main/alloydb-jdbc-connector/src/test/java/com/google/cloud/alloydb/RefresherTest.java).

BTW could you please rename the tests in `RefresherTest` ? Some of them starts with `testCloudSql`

Reply:
I updated `RefresherTest` to remove "CloudSql" from the testcase names. 
I restored some of the test cases to `ConnectionInfoCacheTest`. I'll leave inline comments about why some testcases were modified or removed.

Reply:
testGetConnectionInfo_schedulesNextOperation was removed because it relied on DeterministicExecutor. It is covered by `RefresherTest.testRefreshesExpiredData()`